R/screenshot.R

```diff
@@ -91,18 +103,16 @@ chromote_session_screenshot <- function(
         ymin <- max(ymin, 0)
         ymax <- min(ymax, overall_height)
 
-        self$Page$captureScreenshot(
-          clip = list(
-            x = xmin,
-            y = ymin,
-            width  = xmax - xmin,
-            height = ymax - ymin,
-            scale = scale / private$pixel_ratio
-          ),
-          fromSurface = TRUE,
-          captureBeyondViewport = TRUE,
-          wait_ = FALSE
+        screenshot_args$clip <- list(
+          x = xmin,
+          y = ymin,
+          width  = xmax - xmin,
+          height = ymax - ymin,
+          scale = scale / private$pixel_ratio
         )
+        screenshot_args$wait_ <- FALSE
```

Comment:
Maybe we be allowing these args to be overridden by `options`?

```suggestion
        args <- list(
          wait_ = FALSE,
          clip = list(
            x = xmin,
            y = ymin,
            width  = xmax - xmin,
            height = ymax - ymin,
            scale = scale / private$pixel_ratio
          )
        )
        screenshot_args <- utils::modifyList(args, screenshot_args)
```

Reply:
No, we should set these directly. `wait_ = FALSE` is required by the pattern used in the function and `clip` is determined from user input. If we wanted to give direct access to `clip`, we'd need to handle that in a separate code path, but that's the purpose of the `cliprect` argument so we're covered.